src/WebJobs.Script.Grpc/Startup.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace Microsoft.Azure.WebJobs.Script.Grpc
+{
+    internal class Startup
+    {
+        private const int MaxMessageLengthBytes = 128 * 1024 * 1024;
```

Comment:
This needs to ` public const int MaxMessageLengthBytes = int.MaxValue;`  chaning this size will cause apps that send large to fail. Can you please add unit test so that we dont accidentally update the size in future?

Reply:
Oh sorry, I must have used an outdated version to port. I will update this.

Reply:
Updated and added a test